aii-core/src/main/perl/Shellfe.pm

```diff
@@ -657,7 +657,7 @@ sub fetch_profiles
             next;
         }
         my @err = $fh->fetchProfile();
-        if ($err[0] != SUCCESS) {
+        if (! $err[0] || $err[0] == -1) {
```

Comment:
i sincerely hope we never add `-2`, and that we even get rid of `-1`. failures should be easy to test, the reason why something when wrong can be handled via e.g. a fail `attribute` (even better, retrun undef in case of network issue, and `die()` for everything else).

anyway, i'll use the `$ERROR` readonly that is actually returned (altough the pod says `-1`).